exercises/exercise5.py

```diff
@@ -38,10 +38,43 @@
 from typing import List
 
 
-def calcular_sueldos(contribuyentes: List[Contribuyente]):
-    """Data una lista de contribuyentes, devuelve una lista de los sueldos de
-    cada uno."""
-
+def calcular_sueldos(contribuyentes: List):
+    lista = []
+    monotributista = contribuyentes[0]
+    empleado = contribuyentes[1]
+    lista.append(monotributista.calcular_sueldo())
+    lista.append(empleado.calcular_sueldo())
+
+    return lista
```

Comment:
De esta manera es poco optima. Si se tuviese una lista con mas contribuyentes no funcionaria. La manera correcta es hacer una función mas general, que independientemente del tamaño de la lista y del tipo de contribuyente, retorne todos los sueldos

Algo asi podria ser lo siguiente  : 

def calcular_sueldos(contribuyentes: List[Contribuyente]):
    sueldos=[]
    for i in contribuyentes:
        sueldos.append(i.calcular_sueldo())
    return sueldos